Week06/lesson12/homework/rock_paper_scissors_starter/js/main.js

```diff
@@ -0,0 +1,64 @@
+var botOptions = ["rock", "paper", "scissors"];
+
+var humanScore = 0;
+
+var computerScore = 0;
+
+function humanRock (e) {
+  var humanChoice = $("#rock").attr("id");
+  var botChoice = botOptions[Math.floor(Math.random() * botOptions.length)];
+  console.log(botChoice);
+  compare(humanChoice, botChoice);
+}
+
+function humanPaper (e) {
+  var humanChoice = $("#paper").attr("id");
+  var botChoice = botOptions[Math.floor(Math.random() * botOptions.length)];
+  console.log(botChoice);
+  compare(humanChoice, botChoice);
+}
+
+function humanScissors (e) {
+  var humanChoice = $("#scissors").attr("id");
+  var botChoice = botOptions[Math.floor(Math.random() * botOptions.length)];
+  console.log(botChoice);
+  compare(humanChoice, botChoice);
+}
+
+function compare (humanChoice, botChoice) {
+
+  if (humanChoice === botChoice) {
+
+    $("#status").html("You have chosen <strong>" + humanChoice + "</strong> and.................. the Bot has chosen <strong>" + botChoice + "</strong>:<p>It's a draw!</p>");
+
+  }
+
+  else if ((humanChoice === "rock" && botChoice === "scissors") || (humanChoice === "paper" && botChoice === "rock") || (humanChoice === "scissors" && botChoice === "paper")) {
```

Comment:
Love that you're isolating all the conditions that are "winners" and then all the ones that are "losers", that's very clean, well done